values.yaml

```diff
@@ -62,6 +62,7 @@ master:
     NUMBER_OF_MASTERS: "2"
 
 curator:
+  enable: false
```

Comment:
This technically breaks previous installations, so a note somewhere might make sense about that.

Reply:
Yeah, I think it's better to leave it at the previous implicit default of `true`